### ISSUE TITLE
Typo in stryker init command log about the temp directory name

### ISSUE THREAD

**Issue by xrissot-ibm** [stryker-mutator/stryker#1956]
**Summary**
When you run the init Stryker command at the end you have an info message stating that you have to add the Stryker temp directory to gitignore file:

> Note: Stryker will use `.stryker-temp` as location for temporary files. Be sure to add it to your ignored files in source control.
![image](https://user-images.githubusercontent.com/29564567/71996436-7fb1b280-323c-11ea-97ff-6f817c64aa27.png)

The problem is that the created folder is named stryker-tmp and not stryker-temp:
![image](https://user-images.githubusercontent.com/29564567/71996393-6f99d300-323c-11ea-9232-84e5d071d9ea.png)

**Stryker environment**

```
+-- @stryker-mutator/core@2.4.0
| +-- @stryker-mutator/api@2.4.0
| +-- @stryker-mutator/util@2.4.0
+-- @stryker-mutator/html-reporter@2.4.0
| +-- @stryker-mutator/api@2.4.0 deduped
| +-- @stryker-mutator/util@2.4.0 deduped
+-- @stryker-mutator/jest-runner@2.4.0
| +-- @stryker-mutator/api@2.4.0 deduped
+-- @stryker-mutator/typescript@2.4.0
| +-- @stryker-mutator/api@2.4.0 deduped
| +-- @stryker-mutator/util@2.4.0 deduped

```

```
| +-- jest@24.9.0

```

**Your Environment**

| software         | version(s)
| ---------------- | -------
| node             | v10.15.3
| npm              | 6.4.1
| Operating System | Windows 10

**Comment by simondel** [comment]
Thanks for letting us know! Would you be willing to make this fix yourself?

**Comment by xrissot-ibm** [comment]
> 
> 
> Thanks for letting us know! Would you be willing to make this fix yourself?

Unfortunately I don't have enough bandwidth at the moment to do so :(

**Comment by simondel** [comment]
Fixed with version 2.5.0
